packages/agent/src/orchestrate/analyze/orchestrateAnalyze.ts

```diff
@@ -72,22 +76,18 @@ export const orchestrateAnalyze =
 
     const retryCount = 3 as const;
     const progress = {
-      total: tableOfContents.length * retryCount,
+      total: tableOfContents.length,
```

Comment:
Samchon told me to simply multiply by the number of retries to estimate how many attempts it might take. This was meant as a way to intuitively show the expected time required to the user, assuming the review count is set to the maximum.

Reply:
well.. i understand 
i will revert and left comment it
thank you